bootstrap.sh

```diff
@@ -5,18 +5,40 @@ cd $(dirname "$BASH_SOURCE")
 set -ev
 
 # Update puppet to version >= 3.2.2 before using puppet provisioning.
-wget https://apt.puppetlabs.com/puppetlabs-release-pc1-trusty.deb
-dpkg -i puppetlabs-release-pc1-trusty.deb
+if [ ! -e puppetlabs-release-pc1-trusty.deb ]
+then
+    wget https://apt.puppetlabs.com/puppetlabs-release-pc1-trusty.deb
+fi
+dpkg --install puppetlabs-release-pc1-trusty.deb
 apt-get update
-apt-get -y install puppet-agent
+apt-get --assume-yes install puppet-agent
 export PATH=/opt/puppetlabs/bin:$PATH
 
-cp -f Puppetfile /opt/puppetlabs/puppet/Puppetfile
+DIRECTORY=/opt/mes-aides-bootstrap
+mkdir --parents $DIRECTORY/manifests
+for element in bootstrap ops
+do
+    REPO_PATH=manifests/$element.pp
+    if [ -e $REPO_PATH ]
```

Comment:
I don't understand that disjunction. It seems like you prefer a local element if it's already here, or download it otherwise. But there could be version mismatch. What are you trying to achieve?